### ISSUE TITLE
Neural Network for Video classification

### ISSUE THREAD

**Issue by SergejL** [SergejL/Image-and-Video-context-determination#3]
use label from image classifier to train neural network and determine video class

https://machinelearningmastery.com/tutorial-first-neural-network-python-keras/
